shared/components/ResourceGraph/buildGraph/buildNetworkGraph.js

```diff
@@ -2,47 +2,48 @@ import { wrap, makeNode, makeEdge, makeRank, makeCluster } from './helpers';
 import { match } from './../relations/relations';
 import { findCommonPrefix } from './../../..';
 
-// layers of network graph
-export const networkFlowKinds = [
-  ['VirtualService', 'Gateway'],
-  ['APIRule', 'Function'],
-  'Service',
-  'Pod',
-  ['Secret', 'ConfigMap'],
-  'ServiceAccount',
+const networkFlowLevels = [
```

Comment:
do we really need this? instead of simply going from `Math.min(...configs.map(c => c.networkFlowLevel))` to `Math.max(...options.map(o => o.networkFlowLevel))`?

Reply:
(and ignoring levels without any resources along the way)